frontend/13f-app/src/App.js

```diff
@@ -15,6 +18,8 @@ function App () {
         <Route path='/' exact component= {Home} />
         <Route path='/search' exact component= {SearchResults} />
         <Route path='/company/:companyId' exact component= {SearchResults} />
+        <Route path='/company/:companyId/edgarfiling' exact component= {FilingsPage} />
+        <Route path='/company/:companyId/edgarfiling/:filingId/data' exact component= {DataPage} />
```

Comment:
```suggestion
        <Route path='/company/:companyId/edgar-filing' exact component= {FilingsPage} />
        <Route path='/company/:companyId/edgar-filing/:filingId/data' exact component= {DataPage} />
```

In the interest of readability, I suggest using hyphens to separate resource names in URIs. [Good discussion here](https://stackoverflow.com/questions/10302179/hyphen-underscore-or-camelcase-as-word-delimiter-in-uris)